sospice/catalog/catalog.py

```diff
@@ -356,12 +356,24 @@ def plot_fov(self, ax, **kwargs):
             fovs.reset_index(inplace=True)
         else:
             fovs = Catalog(data_frame=self[list(required_columns)])
+        
+        fovs=fovs.sort_values(by=['DATE'])
+        Nfov=1*len(fovs)
+        if Nfov>12:
+            fovs=fovs[0:12]
+            print("WARNING: The selected catalog contains more observations (%i) than can be displayed. Only the first 12 observations are shown. To show the remaining observations you could select e.g. a slightly different time frame."%(Nfov))
```

Comment:
Maybe the limit could be a soft limit, e.g. having the limit as a parameter of the function with 12 as a default value, allowing the user to change the limit if they want to.